ts/screens/authentication/cie/CieCardReaderScreen.tsx

```diff
@@ -249,9 +295,11 @@ class CieCardReaderScreen extends React.PureComponent<Props, State> {
         <Content bounces={false} noPadded={true}>
           <Text style={styles.padded}>{this.state.subtitle}</Text>
```

Comment:
What do you think if, for the first iteration when we come on this screen we let the screen reader focus on this text?

When we navigate to the reader screen the text says "Appoggia per qualche secondo la tua carta..." I think it would be really helpful for the user to know directly how to interact with this screen.

What do you think?

Reply:
I thought the same, but it seems to don't work.
If you do make it works, a commit it's appreciate :D

Reply:
As discussed I try a solution and it seems to work
https://github.com/pagopa/io-app/pull/2060/commits/1f64b1580b5e99029d3a99063270afca91c56097

Reply:
@CrisTofani 
On CiePinScreen could you test if it could be a solution?
added in the last commit https://github.com/pagopa/io-app/pull/2060/commits/031a9b1df401bfe1b31303080a6721317bee2834